src/cryptonote_core/blockchain.cpp

```diff
@@ -840,6 +841,19 @@ crypto::hash Blockchain::get_block_id_by_height(uint64_t height) const
   }
   return null_hash;
 }
+
+//------------------------------------------------------------------
+void Blockchain::setup_seedhash(const block& b, const uint64_t height) const
+{
+  int cn_variant = b.major_version >= 7 ? b.major_version - 6 : 0;
+  if (cn_variant >= 6) {
+    uint64_t seed_height;
+    if (rx_needhash(height, &seed_height)) {
+      crypto::hash hash = get_block_id_by_height(seed_height);
+      rx_seedhash(seed_height, (char *)&hash, 0);
```

Comment:
hash.data gets you the char[] without a cast fwiw.

Reply:
OK, will change.